FrameworkSystem/Service/SystemAdministratorHandler.py

```diff
@@ -467,7 +467,7 @@ def export_checkComponentLog(self, component):
       startDir = gComponentInstaller.startDir
       currentLog = startDir + '/' + system + '_' + cname + '/log/current'
       try:
-        logFile = file(currentLog, 'r')
+        logFile = open(currentLog, 'r')
```

Comment:
```suggestion
        with open(currentLog, 'r') as logFile:
          logLines = logFile.readlines()
```
and delete the two lines lower down